content/boilerplates/kubectl-multicluster-contexts.md

```diff
@@ -10,9 +12,16 @@
               cluster2   cluster2   user@foo.com   default
     {{< /text >}}
 
-* Export the following environment variables with the context names of your configuration:
+*   Store the context names of your clusters in environment variables:
 
     {{< text bash >}}
-    $ export CTX_CLUSTER1=<KUBECONFIG_CONTEXT_NAME_FOR_CLUSTER_1>
-    $ export CTX_CLUSTER2=<KUBECONFIG_CONTEXT_NAME_FOR_CLUSTER_2>
+    $ export CTX_CLUSTER1=$(kubectl config view -o jsonpath='{.contexts[0].name}')
+    $ export CTX_CLUSTER2=$(kubectl config view -o jsonpath='{.contexts[1].name}')
+    $ echo CTX_CLUSTER1 = ${CTX_CLUSTER1}, CTX_CLUSTER2 = ${CTX_CLUSTER2}
+    CTX_CLUSTER1 = cluster1, CTX_CLUSTER2 = cluster2
     {{< /text >}}
+
+{{< tip >}}
```

Comment:
Maybe this should be indented 4 spaces too, since it's associated with the above step.

Reply:
@frankbu Done in https://github.com/istio/istio.io/pull/4073/commits/6f4f0c9e0cdd9df98f89523ecb36475206d0e7eb.